spec/models/child.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require './spec/models/proxy'
+require 'attr_encrypted'
+
+class Child < PorkyLib::ApplicationRecord
+  include PorkyLib::PartitionProvider
+  include PorkyLib::HasEncryptedAttributes
+  include PorkyLib::HasGuid
+
+  partition_provider :child
+
+  belongs_to :proxy, required: true
+
+  attr_encrypted :value, encryptor: PorkyLib::CachingEncryptor, encrypt_method: :zt_encrypt, decrypt_method: :zt_decrypt,
+                 encode: true, partition_guid: proc { |object| object.generate_partition_guid },
+                 encryption_epoch: proc { |object| object.generate_encryption_epoch }, cmk_key_id: 'alias/zetatango',
+                 expires_in: 5.minutes
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Align the elements of a hash literal if they span more than one line.](https://app.codacy.com/app/Zetatango/porky_lib/pullRequest?prid=4521942)